src/mono/wasm/features.md

```diff
@@ -55,6 +55,14 @@ Because web browsers do not expose direct access to sockets, we are unable to pr
 
 A prominent limitation is that your application must obey `Cross-Origin Resource Sharing` (CORS) rules in order to perform network requests successfully - see [CORS on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for more information.
 
+Since Net 10 Preview 3 the HTTP client supports [streaming HTTP response](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API/Using_readable_streams#consuming_a_fetch_as_a_stream) by default because all evergreen browsers now support it.
+
+This is a breaking change because the `response.Content.ReadAsStreamAsync()` is no longer `MemoryStream` but `BrowserHttpReadStream` which doesn't support synchronous operations like `Stream.Read(Span<Byte>)`. If your code uses synchronous operations, you can disable the feature or copy the stream into `MemoryStream` yourself.
+
+If you need to disable it, you can use `<WasmEnableStreamingResponse>false</WasmEnableStreamingResponse>` or `DOTNET_WASM_ENABLE_STREAMING_RESPONSE` env variable to do it for all HTTP requests.
+
+Or you can use `request.Options.Set(new HttpRequestOptionsKey<bool>("WebAssemblyEnableStreamingResponse"), false);` for individual request.
+
```

Comment:
Does it make sense to unconditionally enable this on well-known APIs?

If you are forced to turn it off for some reason (using a library that makes HttpRequests that read synchronously and which you don't have control of) then you have to turn it off globally, and that means you stop being able to use things like the async enumerable APIs. 

Does it make sense for those APIs to turn the feature on unconditionally in their implementation?

Reply:
I'm thinking about it this way. If the new behavior gives you trouble, you can use the `<WasmEnableStreamingResponse>false</WasmEnableStreamingResponse>` to get to previous behavior.

Also, now you can choose if enabled by default or disabled by default gives you less trouble.

> Does it make sense for those APIs to turn the feature on unconditionally in their implementation?

Those who know enough and have control over it already did in Net8. This PR is "let's make the better default for everyone".

Reply:
Great, thanks for the clarification.